client/src/components/App.js

```diff
@@ -24,6 +30,18 @@ class App extends Component {
     });
   }
 
+  handleDetails(id) {
+    console.log(id);
+    axios
+      .get(`/api/selected/${id}`)
+      .then(property => {
+        this.setState({ details: property.data });
+      })
+      .catch(err => {
+        console.log(err);
```

Comment:
If you have extra time, you could always add in some proper error handling :)

Reply:
I hope returning a div with information is a better way. I have changed.